.github/workflows/package-apisix-rpm-ubi.yml

```diff
@@ -72,6 +72,7 @@ jobs:
 
       - name: stop and uninstall APISIX(-remote) master
         run: |
+          docker exec ubiInstance bash -c 'yum -y install procps-ng'
```

Comment:
this is an existing bug/mistake in CI. the command `pgrep` used in next line will give error "pgrep not found" and the apisix instance won't be shut down.

In further tests, new apisix instance won't run as the existing one is still running. This causes the next tests to pass.